src/main/java/org/broadinstitute/hellbender/tools/exome/copyratio/CopyRatioModeller.java

```diff
@@ -50,10 +49,11 @@ public CopyRatioModeller(final SegmentedGenome segmentedGenome) {
         //set widths for slice sampling of variance and segment-mean posteriors using empirical variance estimate.
         //variance posterior is inverse chi-squared, segment-mean posteriors are Gaussian; the below expressions
         //approximate the standard deviations of these distributions.
-        final double varianceEstimate = data.estimateVariance();
-        final double varianceSliceSamplingWidth = Math.sqrt(2. * varianceEstimate / data.getNumTargets()) / 10.;    //take the width down an order of magnitude to account for inflation by outliers
         final double coverageMin = data.getCoverageMin();
         final double coverageMax = data.getCoverageMax();
+        final double varianceEstimate = data.estimateVariance();
+        final double varianceSliceSamplingWidth = Math.sqrt(2. * varianceEstimate / data.getNumTargets()) / 10.;    //take the width down an order of magnitude to account for inflation by outliers
+        final double varianceMax = Math.abs(coverageMax - coverageMin);
```

Comment:
In principle, it could be even worse.  If I only see a single point near zero, there's still some probability that it was drawn from a distribution with large variance---it's just less likely the larger the variance.

I think this is a reasonable range, but might add some additional checks and tests.